docs/reference/run.md

```diff
@@ -568,6 +569,63 @@ We have four ways to set memory usage:
   </tbody>
 </table>
 
+Kernel memory is fundamentally different than user memory, since it can't be
+swapped out, which makes it possible to DoS the system by consuming too much
+of this precious resource.
+
+Currently, kernel memory includes：
+ - stack pages
```

Comment:
Can you add a blank line before this bullet list, otherwise it won't render as a bullet-list.

You can try out, and check the documentation, using `make docs` and view the generated docs in your webbrowser